javav2/example_code/redshift/Readme.md

```diff
@@ -29,46 +29,72 @@ For prerequisites, see the [README](../../README.md#Prerequisites) in the `javav
 <!--custom.prerequisites.start-->
 <!--custom.prerequisites.end-->
 
+### Get started
+
+- [Hello AWS Redshift](src/main/java/com/example/redshift/HelloRedshift.java) (`describeClustersPaginator`)
+
 ### Single actions
 
 Code excerpts that show you how to call individual service functions.
 
-- [Create a cluster](src/main/java/com/example/redshift/CreateAndModifyCluster.java#L91) (`CreateCluster`)
-- [Delete a cluster](src/main/java/com/example/redshift/DeleteCluster.java#L6) (`DeleteCluster`)
-- [Describe your clusters](src/main/java/com/example/redshift/DescribeClusters.java#L6) (`DescribeClusters`)
-- [Modify a cluster](src/main/java/com/example/redshift/CreateAndModifyCluster.java#L150) (`ModifyCluster`)
+- [Create a cluster](src/main/java/com/example/scenario/RedshiftScenario.java) (`CreateCluster`)
```

Comment:
Why are there no line numbers in the links? Shouldn't WriteMe have added line number?

Reply:
Ran the tool